src/tests/simulation/tests/integration_tests/backfill_tests.rs

```diff
@@ -49,6 +48,13 @@ select
 from p2;
 "#;
 
+async fn kill_cn_and_wait_recover(cluster: &Cluster) {
```

Comment:
None related to this PR, can we move these utils functions including like `kill_cn_and_meta_and_wait_recover` to `integration_tests` mod, so that they can be defined once and used for all tests.